### ISSUE TITLE
"Reset to initial" in ticket edit view does not work

### ISSUE THREAD

**Issue by a-tze** [crs-tools/tracker#154]
The JS link under the state dropdown isn't functional anymore.

**Comment by jjeising** [comment]
Works fine here in Safari and Chrome, do you have a browser, an error message or circumstances?

**Comment by a-tze** [comment]
Firefox 52 on Win and Linux, possibly related error messages in the browser console:

11:17:53.542 Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf blob:https://tracker.c3voc.de/6a51a6fd-6503-4968-82af-17f7d8507db4 blockiert ("default-src https://tracker.c3voc.de").
11:17:53.550 Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("default-src https://tracker.c3voc.de"). Source: call to eval() or related function blocke

**Comment by a-tze** [comment]
I meant version 57, sorry.

**Comment by jjeising** [comment]
Works fine with 57.0.3 on tracker.c3voc.de. CSP errors are almost always related to plugins trying to inject javascript into the page.
Any active plugins? NoScript?

**Comment by a-tze** [comment]
Yes, there is NoScript, but currently on "All scripts allowed". On Linux there was no NoScript. Other users also agreed that the link was infunctional for quite some time.

**Comment by jjeising** [comment]
Further triage concluded the feature works as designed, but the description is ambiguous and may sugges, that the state should be set to the first state a ticket can be in.
